### PR TITLE
Fix typos, grammatical mistakes in doc strings

### PR DESCRIPTION
**Pull request by singularitti**
I must say I do not understand this code very much. The above corrections are just based on a rough guess of how this code would work. Please check before merging.

### CODE REVIEW

File: src/randgendat.jl [hunk 1]
```diff
@@ -47,11 +45,10 @@ function fixpointperms(j::NTuple{N, Int}) where N
 end
 
 """
-  randblock(::Type{T}, dims::NTuple{N, Int}, j::NTuple{N, Int})
-
-Returns a block of size dims and position j by a uniformly distributed random number
-of type T
+    randblock(::Type{T}, dims::NTuple{N, Int}, j::NTuple{N, Int})
 
+Return a block of size `dims` and position `j` by a series of uniformly distributed random numbers
```
Comment (kdomino) [inline src/randgendat.jl:50]: by -> filled with

File: src/symmetrictensor.jl [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
-bln - Int, number of blocks
-datasize - Int, size of data stored (in each direction the same)
-sqr - Bool, is the last block size a same as ordinary's block size
+# Arguments
+- `frame::ArrayNArrays{T,N}`: the raw data.
+- `bls::Int`: the size of ordinary block.
+- `bln::Int`: the number of blocks.
+- `dats::Int`: the size of data stored (the same in each direction).
+- `sqr::Bool`: if the last block is squared.
 """
```
Comment (kdomino) [inline src/symmetrictensor.jl:12]: My suggestion:

if the last block is squared -> if last blocks are squares (N-squares)


Comment:

in matrix form st looks like that
|x1  x2    y1 |
|.....x3    y2 |
|............ y' |
where y1, y2 (these are last blocks) may not be squares. For N-dimensional tensor the analog would not be 
N-arrays.

File: src/symmetrictensor.jl [hunk 3]
```diff
@@ -76,11 +80,13 @@ function issymetric(ar::Array{T, N}, atol::Float64 = 1e-7) where {T <: AbstractF
...
-  frtest(data::ArrayNArrays{T,N})
+    frtest(data::ArrayNArrays{T,N})
 
-Returns assertion error if: all sizes of nullable array not equal, at least
-  some undergiagonal block not null, some blocks (not last) not squared,
-  some diagonal blocks not symmetric.
+Throw an `AssertionError` if:
+- all sizes of nullable array are not equal, or
+- at least some undergiagonal blocks are not null, or
```
Comment (kdomino) [inline src/symmetrictensor.jl:87]: I think we can remove:   at least

File: src/symmetrictensor.jl [hunk 4]
```diff
@@ -76,11 +80,13 @@ function issymetric(ar::Array{T, N}, atol::Float64 = 1e-7) where {T <: AbstractF
...
+- some blocks (not the last one) are not squared, or
```
Comment (kdomino) [inline src/symmetrictensor.jl:88]: some blocks (not the last ones) are not squares (N-squares)

File: src/symmetrictensor.jl [hunk 5]
```diff
@@ -149,17 +154,21 @@ sizetest(dats::Int, bls::Int) =
...
 getblockunsafe(st::SymmetricTensor, mulind::Tuple) = st.frame[mulind...]
 
 
 """
     getblock(st::SymmetricTensor, i::Tuple)
 
-Returns a block from Symmetric Tensor, works for all multi-indices also not sorted
+Return a block from `st`.
+        
+It also works for all multi-indices which are not sorted.
```
Comment (kdomino) [inline src/symmetrictensor.jl:171]: It also works for multi-indices which are not sorted.

**Review by kdomino — COMMENTED** [review]
@singularitti thanks a lot for corrections. I have just added a few suggestions.

**Comment by singularitti** [comment]
Sorry, why is this closed? I have not had time to correct it.

**Comment by kdomino** [comment]
Right. A bit my mistake. So I will correct the few thinks, and then you can go through again.

File: src/symmetrictensor.jl (hunk 2 above)
Comment (singularitti) [reply src/symmetrictensor.jl:12]: Does "if the last few blocks are squares (N-squares)" work?

**Comment by singularitti** [comment]
All suggestions are adopted. By the way, may I ask if there are any introductions about the terminology used in the code, like "blocks" and "block size"? I am a bit confused about how to use it.

**Comment by kdomino** [comment]
The idea was taken from https://arxiv.org/pdf/1301.7744.pdf   , please see  Fig. 3.2.  Eg. 3.3 therein.  I will update a bit more readme to make it more clear.

**Comment by kdomino** [comment]
I updated Readme, please take a look

**Comment by singularitti** [comment]
Thank you!

**Comment by kdomino** [comment]
For our application to Cumulants, see also https://epubs.siam.org/doi/abs/10.1137/17M1149365 or https://arxiv.org/pdf/1701.05420.pdf especially Eq. 15 therein.

**Pull request merged**